src/app/kubernetes-client.service.ts

```diff
@@ -12,4 +13,13 @@ export class KubernetesClientService {
   getZoneList(): Observable<ZoneList> {
     return this.httpClient.get<ZoneList>('appuio-api/apis/appuio.io/v1/zones');
   }
+
+  getZonePermission(): Observable<boolean> {
+    return this.httpClient
+      .post<SelfSubjectAccessReview>(
+        'appuio-api/apis/authorization.k8s.io/v1/selfsubjectaccessreviews',
+        new SelfSubjectAccessReview('list', 'zones', 'appuio.io')
+      )
+      .pipe(map((result) => result?.status?.allowed ?? false));
```

Comment:
Is the first `?` really necessary?
```suggestion
      .pipe(map((result) => result.status?.allowed ?? false));
```